Easydict/objc/ViewController/Window/WindowManager/EZWindowManager.m

```diff
@@ -317,11 +317,26 @@ - (void)showFloatingWindowType:(EZWindowType)windowType
     }
     
     EZBaseQueryWindow *window = [self windowWithType:windowType];
-    
+    EZBaseQueryViewController *queryViewController = window.queryViewController;
     // If text is nil, means we don't need to query anything, just show the window.
     if (!queryText) {
+        BOOL showTips = NO;
+        if (Configuration.shared.disableTipsView) {
+            showTips = NO;
+        } else {
+            if (actionType != EZActionTypeInputQuery) {
+                showTips = YES;
+            }
+            // except ocr
+            if (actionType == EZActionTypeOCRQuery) {
+                showTips = NO;
+            }
+        }
+        [queryViewController showTipsView:showTips];
```

Comment:
If I just want to show query window, `showTips` is NO, we don't need to call `showTipsView`.